.travis.yml

```diff
@@ -6,24 +6,31 @@ language: python
 
 matrix:
     include:
-      - python: 3.5
-        env: TOX_ENV=py35
+        env: TOX_ENV=py35-pytest6
```

Comment:
I think this is your travis failure problem. YAML syntax error

Reply:
Good spot.